### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by TheLastProject** [TheLastProject/Catima#314]
Still happening in 2.2.0. No clue how to trigger this one.

```
java.lang.NullPointerException: 
  at protect.card_locker.LoyaltyCardEditActivity.updateTempState (LoyaltyCardEditActivity.java:146)
  at protect.card_locker.LoyaltyCardEditActivity.updateTempState (LoyaltyCardEditActivity.java:161)
  at protect.card_locker.LoyaltyCardEditActivity.access$000 (LoyaltyCardEditActivity.java:76)
  at protect.card_locker.LoyaltyCardEditActivity$1.onTextChanged (LoyaltyCardEditActivity.java:254)
  at android.widget.TextView.sendOnTextChanged (TextView.java:10955)
  at android.widget.TextView.setText (TextView.java:6380)
  at android.widget.TextView.setText (TextView.java:6180)
  at android.widget.EditText.setText (EditText.java:146)
  at android.widget.TextView.setText (TextView.java:6137)
  at android.widget.TextView.onRestoreInstanceState (TextView.java:6005)
  at android.view.View.dispatchRestoreInstanceState (View.java:20282)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at com.google.android.material.textfield.TextInputLayout.dispatchRestoreInstanceState (TextInputLayout.java:2938)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.ViewGroup.dispatchRestoreInstanceState (ViewGroup.java:4045)
  at android.view.View.restoreHierarchyState (View.java:20260)
  at com.android.internal.policy.PhoneWindow.restoreHierarchyState (PhoneWindow.java:2184)
  at android.app.Activity.onRestoreInstanceState (Activity.java:1195)
  at protect.card_locker.LoyaltyCardEditActivity.onRestoreInstanceState (LoyaltyCardEditActivity.java:192)
  at android.app.Activity.performRestoreInstanceState (Activity.java:1150)
  at android.app.Instrumentation.callActivityOnRestoreInstanceState (Instrumentation.java:1317)
  at android.app.ActivityThread.handleStartActivity (ActivityThread.java:3153)
  at android.app.servertransaction.TransactionExecutor.performLifecycleSequence (TransactionExecutor.java:180)
  at android.app.servertransaction.TransactionExecutor.cycleToPath (TransactionExecutor.java:165)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:142)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:70)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1947)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:214)
  at android.app.ActivityThread.main (ActivityThread.java:7037)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:494)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:965)
```

**Comment by TheLastProject** [comment]
Finally figured it out: this crash happens when rotating Catima in the edit screen.

**Comment by TheLastProject** [comment]
Fixed in 5ddcb0a66841e19d59b68f17df59a4eb1dd432bf.
